docs/guide/08-tools.md

```diff
@@ -122,82 +122,74 @@ Alternately, you can also use the Karma test runner. The Open Web Components rec
 
-Originally developed to work with the Polymer library, the Polymer CLI can handle build duties for a variety of projects. It's not as flexible and extensible as webpack or Rollup, but it requires minimal configuration.
+For more information on the build steps, see the LitElement [Build for production](https://lit-element.polymer-project.org/guide/build) guide.
 
-To build your project with the Polymer CLI, first install the Polymer CLI:
+open-wc also has [Rollup build resources](https://open-wc.org/building/building-rollup.html). 
 
-`npm i -g polymer-cli`
+### Build your project with webpack
 
-Create a `polymer.json` file in your project folder. A simple example would look like this:
+webpack is a powerful build tool with a large ecosystem of plugins. 
 
-```json
-{
-  "entrypoint": "index.html",
-  "shell": "src/myapp.js",
-  "sources": [
-    "src/**.js",
-    "manifest/**",
-    "package.json"
-  ],
-  "extraDependencies": [
-    "node_modules/@webcomponents/webcomponentsjs/bundles/**"
-  ],
-  "builds": [
-    {"preset": "es6-bundled"}
-  ]
-}
-```
+See the open-wc default webpack configuration provides a great starting point for building projects that use lit-html. See their [webpack page](https://github.com/open-wc/open-wc/tree/master/packages/building-webpack) for instructions on getting started. 
 
-This configuration specifies that the app has an HTML entrypoint called `index.html`, has a main JavaScript file (app shell) called `src/myapp.js`. It will produce a single build, bundled but not transpiled to ES5. For details on the polymer.json file, see [polymer.json specification](https://polymer-library.polymer-project.org/3.0/docs/tools/polymer-json) on the Polymer library site.
+### Build considerations  {#build-considerations}
 
-To build the project, run the following command in your project folder:
+If you're creating your own configuration for webpack, Rollup, or another tool, here are some factors to consider:
 
-`polymer build`
+* ES6 to ES5 compilation.
+* Transforming JavaScript modules to other formats for legacy browsers.
+* lit-html template minification.
+* Polyfills.
 
-For more on building with Polymer CLI, see [Build for production](https://polymer-library.polymer-project.org/3.0/docs/apps/build-for-production) in the Polymer library docs.
+For more details on these considerations, see the LitElement [Build for production](https://lit-element.polymer-project.org/guide/build) guide. lit-html has the same requirements as LitElement, except that lit-html requires only the [Template polyfill](#template-polyfill), not the full Web Components polyfills.
 
-### Build your project with webpack
+#### Compilation and module transform {#transpilation-and-module-transform}
 
+You build tools need to compile ES6 features to ES5 for legacy browsers. n general, ES6 is faster than the ES5 equivalent, so try to serve ES6 to browsers that support it.
```

Comment:
Did you flip this around yet? "You" still needs to be "your" and there's a missing "I" for "In".

Reply:
Gah. Fixed a different instance, apparently maimed this one. Sigh.